### PR TITLE
chore: update readme to have inclusive language

### PR DESCRIPTION
**Pull request by schristoff**
Of course my first PR to this repository is a doc change, sorry in advance.  Follow the [guidelines here](https://itconnect.uw.edu/guides-by-topic/identity-diversity-inclusion/inclusive-language-guide/#)

This is a minor nit just to make it a bit more clear.

## Description

...

## Related Issue

Fixes #

Relates to #

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Other (security config, docs update, etc)

## Checklist before merging

- [ ] Test, docs, adr added or updated as needed
- [ ] [Contributor Guide Steps](https://github.com/defenseunicorns/pepr/blob/main/CONTRIBUTING.md#submitting-a-pull-request) followed

### CODE REVIEW

**Review by jeff-mccoy — CHANGES_REQUESTED** [review]
Not opposed to finding an alternative, but "legacy" does not sufficiently capture the word. It has more to do with knowledge only known by a select group of people, and could be legacy or current knowledge passed within a group.

**Comment by schaeferka** [comment]
What about using “organizational “ instead of “legacy”?

File: README.md [hunk 1]
```diff
@@ -11,7 +11,7 @@
...
+Pepr is on a mission to save Kubernetes from the tyranny of YAML, intimidating glue code, bash scripts, and other makeshift solutions. As a Kubernetes controller, Pepr empowers you to define Kubernetes transformations using TypeScript, without software development expertise thanks to plain-english configurations. Pepr transforms a patchwork of forks, scripts, overlays, and other chaos into a cohesive, well-structured, and maintainable system. With Pepr, you can seamlessly transition IT ops legacy knowledge into code, simplifying documentation, testing, validation, and coordination of changes for a more predictable outcome.
```
Comment (cmwylie19) [inline README.md]:
```suggestion
Pepr is on a mission to save Kubernetes from the tyranny of YAML, intimidating glue code, bash scripts, and other makeshift solutions. As a Kubernetes controller, Pepr empowers you to define Kubernetes transformations using TypeScript, without software development expertise thanks to plain-english configurations. Pepr transforms a patchwork of forks, scripts, overlays, and other chaos into a cohesive, well-structured, and maintainable system. With Pepr, you can seamlessly transition IT ops organizational knowledge into code, simplifying documentation, testing, validation, and coordination of changes for a more predictable outcome.
```

Comment (cmwylie19) [reply README.md]: just added a quick suggestion based on the comments in the PR.

**Pull request merged**